e2e-tests/e2eTest.spec.ts

```diff
@@ -125,9 +158,53 @@ void describe('e2e-tests', () => {
 		assert.equal(req.headers.get('content-length'), '0')
 		assert.equal(await req.text(), '')
 	})
+	const expectedResLastHour: Array<Record<string, string | number>> = []
+	timestampsLastHour.forEach((ts, index) => {
+		expectedResLastHour.push({
+			ts: ts.toISOString(),
+			usedBytes: randomVal[index] ?? 0,
+		})
+	})
+	const expectedResLastHourOnomondo: Array<Record<string, string | number>> = []
+	timestampsLastHour.forEach((ts, index) => {
+		expectedResLastHourOnomondo.push({
+			ts: ts.toISOString(),
+			usedBytes: randomVal2[index] ?? 0,
+		})
+	})
+	const expectedResLastDay: Array<Record<string, string | number>> = []
+	timestampsLastDay.forEach((ts, index) => {
+		expectedResLastDay.push({
+			ts: ts.toISOString(),
+			usedBytes: randomVal[index] ?? 0,
+		})
+	})
```

Comment:
use `timestampsLastHour.map` and assign result directly. Will make code segment shorter and types for `expectedResLastHour` do not have to be written.